README.md

```diff
@@ -58,55 +58,143 @@ documentation](https://www.packer.io/docs).
+   a common suffix.  To discover this suffix, find and click one of the
+   `View more details on Cirrus CI` links (bottom of the *Checks* tab in github).
+   Any **Cirrus-CI** task will do, it doesn't matter which you pick.
 
-6. Toward the top of the page, is a button with an arrow, that reads
-   *VIEW ALL TASKS*.  Click this button.
+3. Toward the top of the page, is a button labeled *VIEW ALL TASKS*.
+   Click this button.
 
-7. Look at the URL in your browser, it will use the form
+4. Look at the URL in your browser, it will be of the form
    `https://cirrus-ci.com/build/<big number>`.  Copy-paste (or otherwise
    record in stone) the **big number**, you'll need it for the next step.
 
-6. Go over to whatever other containers/repository needed the image update.
-   Open the `.cirrus.yml` file, and paste the **big number** *prefixed by the
-   letter 'c'*, in place of the value next to `_BUILT_IMAGE_SUFFIX:`.  For
-   example, if the url was `http://.../12345` you would paste in `c12345`
-   as the value for `_BUILT_IMAGE_SUFFIX:`.
+5. Go over to whatever other containers/repository needed the image update.
+   Open the `.cirrus.yml` file, and find the 'env' line referencing the image
+   suffix.  It will likely be named `_BUILT_IMAGE_SUFFIX:` or something similar.
 
-7. Open up a PR with this change, and push it.  Once all tests pass and your
+7. Paste in the **big number** *prefixed by the letter 'c'*.  The *"c*" indicates
+   the images are *cache iamges*.  For example, if the url was `http://.../12345`
+   you would paste in `c12345` as the value for `_BUILT_IMAGE_SUFFIX:`.
+
+8. Open up a PR with this change, and push it.  Once all tests pass and your
    satisfied with the image changes, ask somebody to review/approve both
    PRs for merging.  If you're feeling generous, perhaps provide cross-links
    between the two PRs in comments, for future reference.
 
-8. After all the PRs are merged, you're done.  You may now attend to the little
-   dog begging you for a walk for the last hour.  Hurry!  Little-dogs, do not
-   have big-dog bladders!
+9. After all the PRs are merged, you're done.
+
+
+
+
+## The image-builder image (overview step 1)
+
+Google compute engine (GCE) does not provide a wide selection of ready-made
+VM images for use.  Instead, a lengthy and sophisticated process is involved
+to preparing, properly format, and import external VM images for use.  In order
+to perform these steps within automation, a dedicated VM image is needed which
+itself has been prepared with the necessary incantations, packages, configuration,
+and magic license keys.
+
+For normal day-to-day use, this process should not need to be modified or
+maintained much.  However, on the off-chance that's ever not true, here is
+an overview of the process followed **by automation** to produce the
+*image-building VM image*:
+
+1. Build the container defined in the `ci` subdirectories `ContainerFile`.
```

Comment:
Oh okay now I understand this. Typo confused me: should be "subdirectory's"

Reply:
oops, yes, you are correct.